README.md

```diff
@@ -67,6 +67,8 @@ Here is overview of availability of DNS entries for GPG keys:
 | OpenSuse | :heavy_check_mark:          | :heavy_check_mark: |
 | RPM Fusion | [requested](https://bugzilla.rpmfusion.org/show_bug.cgi?id=5927)        | :x: |
 | Dropbox    | requested        | :x: |
+| [Debian 11 bullseye (stable)](https://software.opensuse.org/download.html?project=home%3Aandykimpe%3Adebian-buster&package=distribution-gpg-keys) | :heavy_check_mark:        | :x: |
+| Ubuntu LTS [launchpad](https://launchpad.net/~andykimpe/+archive/ubuntu/mock) | :heavy_check_mark:        | :x: |
```

Comment:
This table is just for OpenPGP entries provided via DNS+DNSSEC. Debian does not have it. Just leave this untuched.